project/Dependencies.scala

```diff
@@ -66,6 +67,7 @@ object Dependencies {
     "net.logstash.logback" % "logstash-logback-encoder" % Versions.LogstashLogbackVersion,
     "org.codehaus.janino" % "janino" % Versions.JaninoVersion,
     "org.scala-lang" % "scala-reflect" % Versions.ScalaVersion,
+    "org.flywaydb" % "flyway-core" % Versions.FlywayVersion,
```

Comment:
are you using flyway anymore?

Reply:
No we are not longer using flyway. It will be removed